src/VisualStudio/Core/Def/Implementation/DebuggerIntelliSense/DebuggerTextView.cs

```diff
@@ -364,6 +364,22 @@ public void DisconnectFromIntellisenseControllers()
             }
         }
 
+        public void QueuePostLayoutAction(Action action)
```

Comment:
Can these all be implemented as calls to inner? There's some subtle threading code that these were intended to work around.